components/ai_chat/resources/page/components/attachments/index.tsx

```diff
@@ -127,7 +127,7 @@ export function useFilteredItems(search: string) {
     () =>
       conversation.attachmentsDialog === 'history'
         ? // Note: History is only searched for more than 2 characters. This is an upstream limitation.
```

Comment:
nit ditto: HistoryService limitation. Also, I guess it doesn't hurt to still have this here, but do we need it?